### PR TITLE
[WHO-139] Protect Yourself Page and TopNav Component

### PR DESCRIPTION
**Pull request by myiremark**

### CODE REVIEW

**Comment by advayDev1** [comment]
Still blocked on MIT licensed assets.

File: client/ionic/src/pages/ProtectYourself.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        </IonToolbar>
+      </IonHeader>
+      <IonContent>
+        <IonSlides pager={true}>
+          <IonSlide>
+            <IonCard>
+              <IonCardContent className="pb3 tc ph5">
+                <IonImg
+                  className="w-80 center"
+                  src="assets/content/img/protect_yourself/wash_hands.png"
```
Comment (myiremark) [inline client/ionic/src/pages/ProtectYourself.tsx]: do we want kebab case for dir urls too?

Comment (hspinks) [reply client/ionic/src/pages/ProtectYourself.tsx]: that's my preference, too, but again open to other's preferences

Comment (myiremark) [reply client/ionic/src/pages/ProtectYourself.tsx]: done

File: client/ionic/src/App.tsx [hunk 2]
```diff
@@ -32,6 +33,11 @@ const App: React.FC = () => (
       <IonRouterOutlet>
         <Route path="/about" component={About} exact={true} />
```
Comment (myiremark) [inline client/ionic/src/App.tsx]: don't know why gts fix isnt catching/enforcing these differences, perhaps line length issue? @advayDev1

**Comment by myiremark** [comment]
menu

**Comment by myiremark** [comment]
triage

**Comment by myiremark** [comment]
00_wash_hands
01_cover
02_closed_bin
03_wash_hands_frequently
04_avoid_close_contact
05_seek_medical_help

File: client/ionic/src/App.tsx (hunk 2 above)
Comment (advayDev1) [reply client/ionic/src/App.tsx]: which issue specifically?

File: client/ionic/src/pages/ProtectYourself.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  IonSlide,
+  IonCard,
+  IonCardContent,
+  IonImg,
+} from '@ionic/react';
+import TopNav from '../components/TopNav';
+import 'tachyons';
+import { getUserContext } from '../content/userContext';
+
+function useDynamicFlow(id: string) {
```
Comment (advayDev1) [inline client/ionic/src/pages/ProtectYourself.tsx]: make a TODO to me for DRY, sharing this code

File: content/flows/v1/splash.ww.en.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+type: LinearCarousel
+screens:
+  - type: TextImage
+    # TODO: This should not be part of Protect your Health and might be hardcoded in the app.
```
Comment (advayDev1) [inline content/flows/v1/splash.ww.en.yaml]: Comment no longer needed here.

File: client/ionic/src/App.tsx (hunk 2 above)
Comment (myiremark) [reply client/ionic/src/App.tsx]: the protect yourself route is being changed by gts fix.

File: client/ionic/src/pages/ProtectYourself.tsx (hunk 3 above)
Comment (myiremark) [reply client/ionic/src/pages/ProtectYourself.tsx]: I'll also work on that right now

File: client/ionic/src/App.tsx (hunk 2 above)
Comment (advayDev1) [reply client/ionic/src/App.tsx]: I'm not sure what you're talking about.  I see an /about route with no changes?

Comment (myiremark) [reply client/ionic/src/App.tsx]: sorry, it starts on line 36

File: content/flows/v1/splash.ww.en.yaml (hunk 4 above)
Comment (myiremark) [reply content/flows/v1/splash.ww.en.yaml]: done

File: client/ionic/src/pages/ProtectYourself.tsx (hunk 3 above)
Comment (myiremark) [reply client/ionic/src/pages/ProtectYourself.tsx]: done

File: client/ionic/src/App.tsx (hunk 2 above)
Comment (advayDev1) [reply client/ionic/src/App.tsx]: We should look at that separately, regarding line length lint config.

**Pull request merged**